parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -814,6 +814,13 @@ configGPUDrivers() {
     if [ "$OS" = "$UBUNTU_OS_NAME" ]; then
         mkdir -p /opt/{actions,gpu}
         if [ "${CONTAINER_RUNTIME}" = "containerd" ]; then
+            # if target cloud is AzureUSGovernmentCloud, and NVIDIA_DRIVER_IMAGE contains GRID then use the specific 535 NVIDIA driver image
```

Comment:
should augment with the why, until when, etc.  it doesnt explain or the next person when we should revert this.

Reply:
added additional comments